app/components/markdown-block.js

```diff
@@ -1,43 +1,180 @@
+        <div className={cx(classes.wrapper, imgSide === 'top' && classes.topLayout)}>
+          <img className={classes.imageStyleTop} src={imgUrl} alt="Markdown Block" />
+          {textSection}
+        </div>
+      </div>
+    )
+  }
+
   return (
     <div className={cx(classes.markdownBlock, classes[mode], className)} {...otherProps}>
       <div className={classes.wrapper}>
-        <MarkDown className={classes.mdclasses} options={{ overrides: { ActionButton: { component: ActionButton } } }}>
-          {children}
-        </MarkDown>
+        {subject && <h2 className={classes.subject}>{subject}</h2>}
+        {textSection}
       </div>
     </div>
   )
 }
-export default TextBlock
+
+export default MarkdownBlock
 
 const useStylesFromThemeFunction = createUseStyles(theme => ({
   markdownBlock: {
     textAlign: 'center',
     paddingTop: '1rem',
-    paddingBottom: '4rem',
+  },
+  markdownBlockTopImg: {
+    textAlign: 'center',
+    maxWidth: theme.maxPanelWidth,
+    // marginLeft: 'auto',
+    marginRight: 'auto',
+    // paddingTop: '3rem',
+    paddingBottom: '3rem',
+    maxWidth: '20%',
+    display: 'flex',
+    flexDirection: props => (props.imgSide === 'top' ? 'row' : 'row'),
+    // textAlign: 'center',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      flexDirection: props => (props.imgSide === 'top' ? 'column' : 'column'),
+    },
   },
   wrapper: {
+    display: 'flex',
+    flexDirection: props => (props.imgSide === 'top' ? 'column' : 'column'),
+    justifyContent: 'flex-start',
+    maxWidth: theme.maxPanelWidth,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  wrapperWithImage: {
+    display: 'flex',
+    flexDirection: props => (props.imgSide === 'top' ? 'row' : 'row'),
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      flexDirection: props => (props.imgSide === 'top' ? 'row' : 'column'),
+      maxWidth: '80%',
+    },
     maxWidth: theme.maxPanelWidth,
     marginLeft: 'auto',
     marginRight: 'auto',
     whiteSpace: 'pre-line',
   },
+  subject: {
+    fontSize: '2rem',
+    fontFamily: 'Montserrat',
+    fontStyle: 'normal',
+    fontWeight: 700,
+    textAlign: 'left',
+    maxWidth: theme.maxPanelWidth,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: 0,
+  },
+  imageStyle: {
+    aspectRatio: 2 / 2,
+    width: props => (props.imgSide === 'top' ? '25%' : '50%'),
+    height: props => (props.imgSide === 'top' ? '25%' : '80%'),
+    objectFit: props => (props.imgSide === 'top' ? 'cover' : 'unset'),
+    margin: '1rem',
+    borderRadius: '1rem',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      width: props => (props.imgSide === 'top' ? '60%' : '80%'),
+      height: props => (props.imgSide === 'top' ? '60%' : '80%'),
+    },
+  },
+  imageStyleTop: {
+    aspectRatio: 2 / 2,
+    width: '14rem',
```

Comment:
avoid setting the width and height. no need for hieght at all because aspect ration handles it.   Let the image go as wide is the div it is in, just like the text should go as wide as the div it is in.  It will look weird when the parent div is really wide, but that's okay. Let it be the responsibility of the parent.